### PR TITLE
Expereince card section

### LINKED ISSUE #923: Build Work Ex Component for Expereince Page [React]
[Reference](https://www.hugobazin.com)
image

# Modification w.r.t the above reference:

- The color scheme should be the same as followed throughout the current website
-  Instead of the title appearing over the image, put it above the text for simplicity
- Remove the space allocated for hashtags, emoji, and the hyperlink for the case study
- The text should be in bullet points (as in the  current version of the website)
- Instead of using the data given in the current version of the website, try to come up with something new
- Inplace of the images/illustrations, logos of companies should be used
- The data should be dynamically fetched from a variable as done currently in the expereince.js 
- All the images/logos should be placed inside assets/images/workex/

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Fixes #923 

# Work Done:

- Fixed Responsiveness bug in the Experience card section
- Made adjustments to the image sizes

# Relevant Screenshots/Gifs

Screenshot 2021-07-14 at 12 35 29

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -31,7 +31,7 @@ ul {
 .navbar {
     padding: 8px 16px;
     box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.5);
-    position: fixed;
+    position: sticky;
```
Comment (smaranjitghose) [inline src/components/Navbar/Navbar.css:34]: We need the navbar to be visible when we scroll through the page

File: src/pages/Experience/Experience.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+.exp-card .featured-image {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  border-top-left-radius: 13px;
+  border-top-right-radius: 13px;
+}
+
+.card-body .header {
+  color: var(--meta-col);
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.css:56]: Where is the value for this defined in the stylesheet?

**Comment by smaranjitghose** [comment]
Huge improvement over last PR @Anita-ihuman . Kudos to you for that.

## Further Feedback:

- The page is unresponsive for mobile screens. Please try to fix that issue

unresponsive


- Make the size of the image on the card smaller and square in shape so that more space is allocated for the text., Make the company title Bold/Black, shift the duration towards right, use bullet points


UI_update

- The navbar should be visible while scrolling through the website

File: src/pages/Experience/Experience.js [hunk 3]
```diff
@@ -1,11 +1,98 @@
...
+                alt="experience logo"
+              />
+
+              <article className="card-body">
+                <header>
+                  <div className="title">
+                    <h3 className="header">{item.place}</h3>
+                    <p className="meta">
+                      <span className="role">{item.title}</span>
+                      <span className="duration">{item.time}</span>
```
Comment (anushbhatia) [inline src/pages/Experience/Experience.js:80]: float right the date so it won't get stickup with the role tag

**Review by anushbhatia — CHANGES_REQUESTED** [review]
For very small mobile devices it might not have a container so you can add a container from bootstrap using that and then it will be good to go.

image

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Check this resolution and solve the merge conflict then you are go to be merged

**Comment by smaranjitghose** [comment]
- The cards are unresponsive for certain mobile screens as shown above
unrep

- Make the logos square in size

**Pull request closed without merging**